### DISCUSSION TITLE
Changing output name

### DISCUSSION THREAD

**Discussion by rrodroch** [Kinggerm/GetOrganelle#273] [Q&A] [answered]
Dear Kinggerm,

Is there any way to choose/change output name for the assemblies files? When working with a couple or more libraries, it can get pretty messy to organize everything. It would be helpful to change the filenames or at least the fasta header.

I searched through the wiki and didn't find anything regarding this. Did I somehow overlook that or it's not a option right now?

Thank you for valuable software!

**Comment by JianjunJin** [comment, marked as answer]
`--prefix` is the option you need. Please refer to https://github.com/Kinggerm/GetOrganelle/wiki/Usage#main-scripts for more.
